@shared/api/package.json

```diff
@@ -4,6 +4,7 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
+    "@blockaid/client": "^0.25.0",
```

Comment:
I'm adding the lib as a dep strictly for the types. I thought it'd be nice to not have to worry about updating types with subsequent releases. The cost here is the added bundle size, but imo not a major concern with an extension